python-squid/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -1848,28 +1848,42 @@ public void generator_expression() {
   public void tuples() {
     Tuple empty = parseTuple("()");
     assertThat(empty.getKind()).isEqualTo(Tree.Kind.TUPLE);
+    assertThat(empty.firstToken().value()).isEqualTo("(");
+    assertThat(empty.lastToken().value()).isEqualTo(")");
```

Comment:
IMO it is better to check that first token isSameAs leftParenthesis. Same for lastToken and rightParenthesis